lib/connectors/createsend.rb

```diff
@@ -0,0 +1,38 @@
+module Connectors
+  class Createsend
+    def self.call(params:, auth:)
+      new(params, auth)
+    end
+
+    def initialize(params, auth)
+      @params = params
+      @auth   = auth
+    end
+
+    attr_reader :params, :auth
+    private :params, :auth
+
+    def connector_code
+      "createsend"
+    end
+
+    def description
+      "createsend mailinglist"
+    end
+
+    def account_identifiers
+      {
+        #there is nothing giving by omniauth that we could use
+      }
+    end
+
+    def credential_details
+      {
+
+        refresh_token: auth.credentials.refresh_token,
```

Comment:
not so sure i like that @auth  is the class variable. i would have put  explicitly @refresh_token, @secret_token, @token_expires_at